### ISSUE TITLE
The Windows Installer BREAKS windows!! (3.11.2-win64)

### ISSUE THREAD

**Issue by MxMaster** [sqlitebrowser/sqlitebrowser#1897]
Details for the issue
--------------------

Install the application using the .msi installer version: 3.11.2-win64

#### What did you expect to see?
My VS 2015 and pretty much EVERY executable to NOT stop working for some reason.
VS does not compile/clean/start/debug anything anymore.
Even Notepad++ cannot be launched.
Every so often some error comes up that says this .exe is not a win32 application.
Same goes for notepad++.
As if none of the .exe files on the computer are win32 applications anymore. As if the installer breaks something very fundamental in windows, which stops windows from launching .exe files.

#### What did you see instead?
Many of the .exe files stop working. VS does not compile/run/clean.
Norepad++ does not launch and says this is not a win32 application. Many other applications show the same behavior. This is not limited to VS and Notepad++. Those are just my development tools. But errors as random as Adobe something something is not a win32 application pop up left and right.

Useful extra information
-------------------------
Restoring windows to the restore point that was thankfully created upon installation fixes the issue.

The info below often helps, please fill it out if you're able to. :)

#### What operating system are you using?

- [ ] Windows: ( _version:7 ___ )
- [ ] Linux: ( _distro:_ ___ )
- [ ] Mac OS: ( _version:_ ___ )
- [ ] Other: ___

#### What is your DB4S version?
What's a DB4S?
- [ ] 3.11.2
- [ ] 3.11.1
- [ ] 3.10.1
- [ ] Other: ___

#### Did you also

- [No] Try out the latest nightly version: https://github.com/sqlitebrowser/sqlitebrowser#nightly-builds
- [Yes] Search for an existing similar issue: https://github.com/sqlitebrowser/sqlitebrowser/issues?utf8=%E2%9C%93&q=is%3Aissue%20

Similar issues do exist, but none report the issue as I am. This is alot more fundamental than something breaking. The entire Windows is broken.

**Comment by justinclift** [comment]
Ouch.  Well, that's definitely not supposed to happen (obviously). :frowning_face: 

Just so we have a clearer picture to work with, which version of Windows are you running, and which installer (exactly) did you use?

We recently updated our PortableApp one, which was **supposed** to fix a missing dependency (MSVC 2013 redistributable).  But I'm not sure which installer you grabbed, so it's a bit hard to tell just yet what to investigate. :confused:

**Comment by MxMaster** [comment]
Hmmm...  Well, I picked this one up:
https://download.sqlitebrowser.org/DB.Browser.for.SQLite-3.11.2-win64.msi

But as I'm banging my head over this laptop in front of me, I am guessing this might be the work of VS 2017 and not your installer. Should I close this issue and reopen it later if I became sure?

**Comment by justinclift** [comment]
Yeah.  It **is** more likely it wasn't that particular .msi installer.

But I won't claim "impossible" as all kinds of weird, unexpected interactions can happen with systems. :wink:

Close it for now, and if it does turn out to be something wrong with our installer then you're definitely welcome (encouraged) to re-open so it can be looked into. :smile:

**Comment by sky5walk** [comment]
I just downloaded and ran the latest x64 msi nightly with no errors. Yes, Notepad++ works too.

**Comment by MxMaster** [comment]
Okay, but just so you know, I was developing a project on VS 15 on System.Data.SQLite. And one database file had some empty rows, so I said, let's download a sqlite browser and see the db for myself. And it was only after installing DB browser that everything stopped working. But as I'm too tired to get into this further, I'm just gonna close it. I'll reopen it if I ever see it again.

**Comment by chrisjlocke** [comment]
I've got VS2017 and VS2019 on this laptop, and run the DB4S .MSI installer regularly with no issues.
Windows 7 is fairly solid (albeit old) so can just assume something got wacky in the registry somehow.  I know Win 10 wibbles a bit now and then (I lost the ability to run cmd.exe the other night) so these things 'happen'.  Its not always down to the installer, despite that being the last action you performed - its just the last thing that happened before the wibble.

As this is the only report of the MSI installer corrupting Windows (despite many downloads) I'd say it was 'one of those things', which doesn't help you in the slightest, I know.

As your bum has been nibbled, I hope you've got a system image (using something like Aomei Backupper or Macrium Reflect) before something really chomps on your buttocks.  I downloaded a nasty virus the other week (my own fault) which caused all manner of problems...
